src/components/calcite-action-group/calcite-action-group.tsx

```diff
@@ -7,18 +10,72 @@ import { Component, Host, h, VNode } from "@stencil/core";
 })
 /**
  * @slot - A slot for adding a group of `calcite-action`s.
+ * @slot menu-actions - a slot for adding an overflow menu with actions inside a dropdown.
  */
 export class CalciteActionGroup {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * Indicates whether widget is expanded.
+   */
+  @Prop({ reflect: true }) expanded = false;
+
+  /**
+   * 'Options' text string for the actions menu.
```

Comment:
Nitpick: Maybe avoid using the actual text in the doc in case it changes later on. One less thing to update.